src/adapters/refract/transformSections.coffee

```diff
@@ -39,16 +39,18 @@ module.exports = (parentElement) ->
 
       description = getDescription(element)
 
-      # Then create a new section in the Application AST
-      # corresponding to the Category element.
-      resourceGroup = new blueprintApi.Section({
-        name: _.chain(element).get('meta.title', '').contentOrValue().value()
-        description: description.raw
-        htmlDescription: description.html
-        resources: transformResources(element)
-      })
+      classes = _.get(element, 'meta.classes', [])
```

Comment:
Also, you can only include what you want to match.

```js
check({
  meta: { classes: ['resourceGroup'] }
});
```

This will return `true`.